src/platform/user/profile/vet360/util/transactions.js

```diff
@@ -58,19 +58,19 @@ export const LOW_CONFIDENCE_ADDRESS_ERROR_CODES = new Set([
 export const DECEASED_ERROR_CODES = new Set(['VET360_MVI300']);
 
 export function isPendingTransaction(transaction) {
-  return PENDING_STATUSES.has(transaction.data.attributes.transactionStatus);
+  return PENDING_STATUSES.has(transaction?.data.attributes.transactionStatus);
```

Comment:
The four changes in this file were not strictly required. But I was tired of seeing a bunch of instances where we had to first check to see that a `transaction` was truthy before passing it to these helpers. I also changed a bunch of uses of the `isPendingTransaction` and `isSuccessfulTransaction` helpers to stop doing a truthy check.